package.json

```diff
@@ -2,19 +2,19 @@
   "name": "binode",
   "version": "0.0.0-semantically-released",
   "description": "Allows you to specify a --bin flag for node",
-  "main": "bin.js",
-  "type": "commonjs",
   "scripts": {
-    "test": "node test"
+    "test": "node test/index.mjs && node test/index.js"
   },
   "keywords": [],
   "author": "Kent C. Dodds <me@kentcdodds.com> (https://kentcdodds.com)",
   "license": "MIT",
   "bin": {
-    "binode": "bin.js"
+    "binode": "bin.cjs",
+    "binode-esm": "bin.mjs"
```

Comment:
Could be clever to simply tease out which module system is being used and proceed from there. Seems like that could be doable I suppose by checking `import.meta` or __dirname. Haven't investigated